upup/pkg/fi/cloudup/gce/instancegroups.go

```diff
@@ -62,6 +62,22 @@ func (c *mockGCECloud) DeleteInstance(i *cloudinstances.CloudInstanceGroupMember
 	return recreateCloudInstanceGroupMember(c, i)
 }
 
+// SetTerminationPolicy is used to set the termination policy in a CloudInstanceGroup
+func (c *mockGCECloud) SetTerminationPolicy(*kops.Cluster, string, []cloudinstances.TerminationPolicy) error {
```

Comment:
We are going to need to get this implemented.  It is ok that it is not supported by this PR.  Any idea if this functionality is supported?

Reply:
Admittedly I don't know too much about GCE .. I searched through the API and documentation but couldn't see anything similar to a AWS termination policy on GCE .. For the moment, i just blocked the user from choosing the strategy if your not sunning in AWS.